src/WebJobs.Script.WebHost/Security/KeyManagement/ScriptSecrets.cs

```diff
@@ -13,7 +13,6 @@ public abstract class ScriptSecrets
     {
         protected ScriptSecrets()
         {
-            HostName = ScriptSettingsManager.Instance.GetSetting(EnvironmentSettingNames.AzureWebsiteHostName);
```

Comment:
No longer defaulting this on construction. Setting this at persist time

Reply:
👍